README.md

```diff
@@ -10,21 +46,20 @@ This Lava repository currently supports the following constraint optimization pr
 
```

Comment:
Please update redaction so that it becomes clear is the solution to these problems which is enabled and not the API for the problems (which is enabled but less relevant)